src/main/java/net/geforcemods/securitycraft/datagen/BlockTagGenerator.java

```diff
@@ -257,10 +297,16 @@ protected void registerTags()
 				SCContent.REINFORCED_GREEN_STAINED_GLASS.get(),
 				SCContent.REINFORCED_RED_STAINED_GLASS.get(),
 				SCContent.REINFORCED_BLACK_STAINED_GLASS.get());
+		getOrCreateBuilder(BlockTags.INFINIBURN_OVERWORLD).add(SCContent.REINFORCED_NETHERRACK.get());
+		getOrCreateBuilder(BlockTags.NYLIUM).addTag(SCTags.Blocks.REINFORCED_NYLIUM);
+		getOrCreateBuilder(BlockTags.PRESSURE_PLATES).addTag(SCTags.Blocks.REINFORCED_PRESSURE_PLATES);
 		getOrCreateBuilder(BlockTags.RAILS).add(SCContent.TRACK_MINE.get());
 		getOrCreateBuilder(BlockTags.SLABS).addTag(SCTags.Blocks.REINFORCED_SLABS).add(SCContent.CRYSTAL_QUARTZ_SLAB.get());
 		getOrCreateBuilder(BlockTags.STAIRS).addTag(SCTags.Blocks.REINFORCED_STAIRS).add(SCContent.STAIRS_CRYSTAL_QUARTZ.get());
+		getOrCreateBuilder(BlockTags.STRIDER_WARM_BLOCKS).add(SCContent.FAKE_LAVA_BLOCK.get());
 		getOrCreateBuilder(BlockTags.SIGNS).addTag(SCTags.Blocks.SECRET_SIGNS);
+		getOrCreateBuilder(BlockTags.SOUL_FIRE_BASE_BLOCKS).add(SCContent.REINFORCED_SOUL_SOIL.get());
+		getOrCreateBuilder(BlockTags.SOUL_SPEED_BLOCKS).add(SCContent.REINFORCED_SOUL_SOIL.get());

```

Comment:
Are there equivalents for Soul Sand? If so, please add.

Reply:
There is no reinforced soul sand as far as I'm concerned, and I won't add them because then you have to mess around with bubble columns. Same reasoning for reinforced magma blocks btw.